tools/shared/src/main/scala/org/scalajs/core/tools/io/VirtualFiles.scala

```diff
@@ -204,43 +219,6 @@ trait VirtualFileContainer extends ScalaJSIRContainer {
    */
   def listEntries[T](p: String => Boolean)(
       makeResult: (String, InputStream) => T): List[T]
-
-  /** All the `*.sjsir` files in this container.
-   *
-   *  It is up to the implementation whether these files are read lazily or not.
-   *  The default implementation reads them into memory.
-   *
-   *  Depending on the implementation, calling `sjsirFiles` might be more
-   *  efficient than using `listEntries` with a predicate
-   *  `_.endsWith(".sjsir")`.
-   */
-  def sjsirFiles: List[VirtualRelativeScalaJSIRFile] = {
-    listEntries(_.endsWith(".sjsir")) { (relPath, stream) =>
-      val file = new EntryIRFile(path, relPath)
-      file.content = IO.readInputStreamToByteArray(stream)
-      file.version = version
-      file
-    }
-  }
-
-  def jsFiles: List[VirtualJSFile with RelativeVirtualFile] = {
-    listEntries(_.endsWith(".js")) { (relPath, stream) =>
-      val file = new EntryJSFile(path, relPath)
-      file.content = IO.readInputStreamToString(stream)
-      file.version = version
-      file
-    }
-  }
```

Comment:
Just to double check: This was dead-code?

Reply:
Yes, it was dead code.